es-ds-components/components/es-form-textarea.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+// Prevents attributes from being applied to first <div>
+// v-bind="$attr" is on the input instead
+defineOptions({
+    inheritAttrs: false,
+});
+
+const props = defineProps({
```

Comment:
Nit: the props are only used in the template, so you don't need to save the variable. You can instead just call `defineProps({...` and reference the props without `props.` in the template

Reply:
I had that originally but after adding `defineModel` it made the props have this error:
```
Property 'required' does not exist on type 'CreateComponentPublicInstanceWithMixins<ToResolvedProps<__VLS_PublicProps, { "update:modelValue": (modelValue: string) => any; }>, { model: ModelRef<string | undefined, string, string | undefined, string | undefined>; hasSuccess: () => boolean; hasMessage: () => boolean; hasError: () => boolean; }, ... 23 more ..., ...'.ts-plugin(2339)
```

Reply:
Hmm I'm not seeing that error. In any case not a big deal, don't wanna hold up merging this if it's hard to address

Reply:
Removed! It must be an extension I have installed.